Lib/test/test_typing.py

```diff
@@ -123,38 +123,56 @@ def test_any_works_with_alias(self):
         typing.IO[Any]
 
 
-class NoReturnTests(BaseTestCase):
+class BottomTypeTests:
```

Comment:
I would rather name this to emphasize it's a mixin class; I was for a while mystified by the presence of an uninitialized bottom_type variable.

Reply:
Good call, I renamed it.